02_getting_started.asciidoc

```diff
@@ -36,33 +36,48 @@ If users decide to outsource some functionality to a third party they usually gi
 
 Finally, those seeking simplicity and convenience, even at the expense of control and security, may choose a custodial Lightning wallet. This is the least technically challenging option, but it _undermines the trust model of cryptocurrency_ and should therefore be considered only as a stepping stone towards more control and self-reliance.
 
-In <<lnwallet-categories>> we see the three broad categories of lightning wallets and the relative degree of control they offer to the user.
+There are many ways wallets can be characterized or categorized. 
+From all features the two most important criteria are LN Node use and fund custody. Correspondingly the two most important questions to ask about a specific wallet are:
```

Comment:
I think a third question is something along the lines of where it gets its Bitcoin blockchain data from. Even if you have a "full" LN wallet, and control your own keys, if you blindly accept bogus data from a 3rd party, then you can actually be using "fake coins" without knowing it.

Reply:
Added the question. See Commit # 2.